service.onmute/settings.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of OnMute
+#
+# OnMute is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OnMute is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with OnMute.  If not, see <http://www.gnu.org/licenses/>.
+
+import xbmcaddon
```

Comment:
Not sure if this whole "Script to open settings" is a little overkill?
Someone else please comment on that.

In case it's fine, this...
```python
import xbmcaddon
xbmcaddon.Addon().openSettings()
```
...is really all you need